demo/api/src/importer/decorators/csv-column.decorator.ts

```diff
@@ -0,0 +1,76 @@
+import { Transform } from "class-transformer";
+import { parse, parseISO } from "date-fns";
+
+import { ImporterInputClass } from "../importer-input.type";
+
+export enum Type {
```

Comment:
Naming: I'd use something more specific since class-transform, MikroORM and NestJS already have a Type type. Maybe CsvColumnType?

Reply:
Sounds good